runtime/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -81,4 +84,26 @@ void regmodule_offline_transformations(py::module m) {
         },
         py::arg("function"),
         py::arg("param_res_names"));
+
+    m_offline_transformations.def(
+        "compress_model_transformation",
+        [](std::shared_ptr<ov::Function> function) {
+            ngraph::pass::Manager manager;
+            manager.register_pass<ov::pass::MarkPrecisionSensitiveSubgraphs>();
+            manager.register_pass<ov::pass::CompressFloatConstants>();
+            manager.run_passes(function);
+        },
+        py::arg("function"));
+
+    // todo: remove as serialize as part of passManager api will be merged
+    m_offline_transformations.def(
+        "serialize",
+        [](std::shared_ptr<ov::Function> function, const std::string& path_to_xml, const std::string& path_to_bin) {
+            ngraph::pass::Manager manager;
+            manager.register_pass<ngraph::pass::Serialize>(path_to_xml, path_to_bin);
```

Comment:
this mistake also is in https://github.com/openvinotoolkit/openvino/pull/8301
Please, use `ov::pass` instead of `ngraph::pass`